src/github.com/matrix-org/dendron/dendron/main.go

```diff
@@ -148,49 +125,62 @@ func main() {
 		}
 	}
 
-	var synapse *exec.Cmd
-	var pusher *exec.Cmd
-	var synchrotron *exec.Cmd
-
 	var synapseLog = log.WithField("synapse", synapseURL.String())
 
+	// Used to terminate dendron.
+	terminate := make(chan string, 1)
+
+	signals := make(chan os.Signal, 1)
+	signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM)
```

Comment:
FTR these signals already trigger run-time panics (provided you don't `signal.Notify`). If that panic is on the "main" goroutine, it would mean that all your `defer` calls will fire anyway. Are you sure we need to listen for these signals?